<ui version="4.0" >
 <class>Chapters</class>
 <widget class="QWidget" name="Chapters" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Chapters</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="chaptersLabel" >
       <property name="text" >
        <string>&amp;Chapters:</string>
       </property>
       <property name="buddy" >
        <cstring>chaptersListView</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="chaptersView" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>275</width>
         <height>0</height>
        </size>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights" >
        <bool>true</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KPushButton" name="addButton" >
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="removeButton" >
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Remove</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="KMFImageView" native="1" name="video" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>75</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="customPreviewButton" >
       <property name="text" >
        <string>&amp;Use as Custom Preview</string>
       </property>
       <property name="shortcut" >
        <string>Alt+U</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="KPushButton" name="startButton" >
           <property name="toolTip" >
            <string>Start</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="rewButton" >
           <property name="toolTip" >
            <string>Rewind</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="prevButton" >
           <property name="toolTip" >
            <string>Previous frame</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="nextButton" >
           <property name="toolTip" >
            <string>Next frame</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="fwdButton" >
           <property name="toolTip" >
            <string>Forward</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="endButton" >
           <property name="toolTip" >
            <string>End</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel" >
         <property name="text" >
          <string>00:00</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="timeSlider" >
         <property name="maximum" >
          <number>65534</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KMFImageView</class>
   <extends>QWidget</extends>
   <header>kmfimageview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>rewButton</tabstop>
  <tabstop>prevButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>fwdButton</tabstop>
  <tabstop>endButton</tabstop>
  <tabstop>timeSlider</tabstop>
 </tabstops>
 <includes>
  <include location="local" >kmfimageview.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
